modules/standard/Random.chpl

```diff
@@ -80,7 +80,12 @@ module Random {
 
     // Will be deprecated in 1.14.
     pragma "no doc"
-    const SeedGenerator: SeedGeneratorsDeprecated;
+    const SeedGenerator_actual: SeedGeneratorsDeprecated;
+    pragma "no doc"
+    proc SeedGenerator {
+      compilerWarning("Random.SeedGenerator is deprecated - use Random.SeedGenerators");
+      return SeedGenerator_actual;
+    }
```

Comment:
I don't think this is worth changing at this point since it's code that's going to go away anyway, but this compilerWarning() could've potentially been put into a constructor for SeedGeneratorDeprecated as well to avoid the need for the proc + _actual variable.  (Or into its currentTime query on the argument that any meaningful use of SeedGenerator would call the routine?  even though that's not as bulletproof...)